wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -62,10 +77,11 @@ public static HPCCWsWorkUnitsClient get(Connection connection)
 
     public static final String          WSWORKUNITSWSDLURI          = "/WsWorkunits";
     private WsWorkunitsServiceSoapProxy wsWorkunitsServiceSoapProxy = null;
+    private org.hpccsystems.ws.client.gen.wsworkunits.v1_56.WsWorkunitsServiceSoapProxy fallBackWorkunitsServiceSoapProxy =null;
```

Comment:
Yes that should be completely backwards compatible, It might send more parameters than an older esp is able to recognize, but so long as the parameter names don't change we don't have issues.  the esp will just throw out unknowns.